spec/spec_tests/crud_spec.rb

```diff
@@ -5,18 +5,8 @@
 
 require 'runners/crud'
 
-describe 'CRUD v1 spec tests' do
+describe 'CRUD spec tests' do
```

Comment:
The `v1` is meaningful here, I suggest keeping it.

Reply:
done